### PR TITLE
Setup pre commit

### PR DESCRIPTION
**Pull request by jaydhulia**

### CODE REVIEW

**Review by drbalar — APPROVED** [review]
Works for me

File: .github/workflows/pre-commit-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: pre-commit
+
+on: [push, pull_request]
```
Comment (ahmedhamodi) [inline .github/workflows/pre-commit-workflow.yml:3]: What other options are there?

**Comment by jaydhulia** [comment]
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#webhook-events
The ones I added are the only relevant ones for us on workflow.


On Mar 13, 2021, at 5:17 PM, Ahmed Hamodi ***@***.***> wrote:

﻿

@ahmedhamodi commented on this pull request.

________________________________

In .github/workflows/pre-commit-workflow.yml<https://github.com/uwblueprint/cdc-frontend/pull/11#discussion_r593818962>:

> @@ -0,0 +1,27 @@
+name: pre-commit
+
+on: [push, pull_request]


What other options are there?

File: .github/workflows/pre-commit-workflow.yml (hunk 1 above)
Comment (jaydhulia) [reply .github/workflows/pre-commit-workflow.yml:3]: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#webhook-events
The ones I added are the only relevant ones for us on workflow. @ahmedhamodi

**Pull request merged**